sources/api/schnauzer/src/v1.rs

```diff
@@ -157,6 +157,168 @@ pub fn build_template_registry() -> Result<handlebars::Handlebars<'static>> {
     Ok(template_registry)
 }
 
+pub mod cli {
+    //! CLI module for schnauzer v1
+    use snafu::{ensure, OptionExt, ResultExt};
+    use std::collections::HashMap;
+    use std::string::String;
+    use std::{env, process};
+
+    // Setting generators do not require dynamic socket paths at this moment.
```

Comment:
nit: I don't understand this comment's relevance even though it was in the original also